pkg/util/k8sutil/k8sutil.go

```diff
@@ -37,6 +37,14 @@ func Int32ToPointer(value int32) *int32 {
 	return &value
 }
 
+func UInt32ToPointer(value uint32) *uint32 {
+	return &value
+}
+
+func StringToPointer(str string) *string {
```

Comment:
Please note, that `k8s.io/utils/pointer/pointer.go` package already contains these kind of functions.

Reply:
Ah good to know :smiley: